canonical-web-and-design/juju.is
# Render the discourse content on the homepage of docs

## Done
- Removed the custom docs homepage

## QA
- Go to /docs
- See the content matches the content from the discourse home topic
- Check search still works. _If you need the key ask me_

Fixes https://github.com/canonical-web-and-design/juju.is/issues/57
